junit-tests/src/test/java/org/junit/gen5/engine/DummyTestEngineDescriptor.java

```diff
@@ -10,20 +10,21 @@
 
 package org.junit.gen5.engine;
 
-class DummyTestDescriptor extends AbstractTestDescriptor implements Leaf<DummyEngineExecutionContext> {
-
-	public static final String ENGINE_ID = "dummy";
-
+class DummyTestEngineDescriptor extends AbstractTestDescriptor implements Leaf<DummyEngineExecutionContext> {
```

Comment:
Why did you rename this class? I think the new name is confusing because it's not actually an `EngineDescriptor`.